src/components/common.js

```diff
@@ -75,7 +75,7 @@ export const buttonPrimary = ({ disabled, ...props }, children) => {
   }, props), children)
```

Comment:
Hyperscript can't handle single react elements as children, so certain things aren't working (e.g. the terminal link). In many of these components we can now just let `children` pass through with the rest of the props. When that's not possible, we'll need to pass it as an array, e.g. `[children]`

Reply:
Ah yes, I didn't test this morning because dev was unhappy, so I was expecting _something_ to be weird.

Reply:
Oh, wait, I should just let them pass implicitly, yes.